Auth0/OAuth2Grant.swift

```diff
@@ -119,15 +182,16 @@ struct PKCE: OAuth2Grant {
         components.a0_queryValues.forEach { items[$0] = $1 }
         return items
     }
+
 }
 
-private func validate(responseType: [ResponseType], token: String?, nonce: String?) -> Bool {
-    guard responseType.contains(.idToken) else { return true }
-    guard
-        let expectedNonce = nonce,
-        let token = token
-        else { return false }
-    let claims = decodeJwt(token)
-    let actualNonce = claims?["nonce"] as? String
-    return actualNonce == expectedNonce
+private func validate(idToken: String?,
```

Comment:
add a comment: this method will skip the validation if the response type does not contain "id_token"

Reply:
Added in ca28451be12118c8928745aead45feaa63e669e1.